packages/ui-voip/src/views/CallHistoryContextualbar/CallHistoryContextualbar.tsx

```diff
@@ -64,14 +66,15 @@ const CallHistoryContextualBar = ({ onClose, actions, contact, data }: CallHisto
 
 	const { voiceCall, directMessage } = actions;
 	const { duration, callId, direction, startedAt } = data;
+	const { state } = useMediaCallExternalContext();
 
 	const date = useFullStartDate(startedAt);
 	return (
 		<ContextualbarDialog onClose={onClose}>
 			<ContextualbarHeader>
 				<ContextualbarIcon name='info-circled' />
 				<ContextualbarTitle>{t('Call_info')}</ContextualbarTitle>
-				<CallHistoryActions onClose={onClose} actions={actions} />
+				<CallHistoryActions onClose={onClose} actions={actions} state={state} />
```

Comment:
Why drilling the prop here instead of calling `useMediaCallExternalContext` inside `CallHistoryActions`?